articles/uploading-images-to-cloudinary-from-django-application/index.md

```diff
@@ -0,0 +1,270 @@
+Sometimes while working on applications that require the use of images, a simple process like uploading images to a server can become difficult.
+If you deploy your Django to platform like heroku, you can't save image.
+
+In this tutorial, we look at how to save images in Django using CLoudinary.
+
+Cloudinary is an end-to-end image- and video-management solution for websites and mobile apps, covering everything from image and video uploads, storage, manipulations, optimizations to delivery.
+
+### Prerequisites
+
+To follow along with this tutorial, you need Python3 installed on your machine.
+
+### Creating and setting up a new Django project
```

Comment:
It would be great, if you could specify an objective of what this tutorial is going to be about or what are you going to build